src/ape/api/networks.py

```diff
@@ -509,6 +509,18 @@ def enrich_calltree(self, call: CallTreeNode, **kwargs) -> CallTreeNode:
         """
         return call
 
+    @abstractmethod
```

Comment:
adding an abstract method is technically a breaking chance as it requires plugin to have implemented it.
to get around this, you can use `@raises_not_implemented` or come up with a really generic / basic implementation.